demo/hover.html

```diff
@@ -0,0 +1,139 @@
+<!doctype html>
```

Comment:
@samikeijonen you'll need to version bump the package.json so we can publish as well